src/input/mod.rs

```diff
@@ -2379,7 +2461,8 @@ impl State {
                     .unwrap()
                     .toggle_stacking_focused(seat);
                 if let Some(new_focus) = res {
-                    Shell::set_focus(self, Some(&new_focus), seat, Some(serial));
+                    //TODO: verify whether cursor follows focus should be checked when toggling stacking
+                    Shell::set_focus(self, Some(&new_focus), seat, Some(serial), false);
```

Comment:
Drop the TODO please. I don't think it should, the window / group won't visually move when creating a stack, so if the user moved the pointer outside of it, that was probably intentional.